Library/Formula/task.rb

```diff
@@ -23,4 +23,12 @@ def install
   test do
     system "#{bin}/task", "--version"
   end
+
+  def caveats; <<-EOS.undent
+    In order to use the completions for the fish shell, extend $fish_complete_path with:
+      #{HOMEBREW_PREFIX}/share/doc/task/scripts/fish/
+    e.g. by putting this line into ~/.config/fish/config.fish:
+      set -x fish_complete_path #{HOMEBREW_PREFIX}/share/doc/task/scripts/fish/ $fish_complete_path
+    EOS
+  end
```

Comment:
I just think that most people using Task aren't using Fish. I'd suggest that it be added more prominent in the Task documentation.